creator-node/src/components/replicaSet/URSMRegistrationComponentService.js

```diff
@@ -173,6 +150,71 @@ const respondToURSMRequestForSignature = async ({ libs: audiusLibs, nodeConfig }
   }
 }
 
+const getSPInfo = async ({ audiusLibs, spID, reqTimestamp, reqSignature }) => {
```

Comment:
don't think we need timestamp or signature here?

Reply:
yeah, we don't need these for the sake of this fn. I can move these checks outside into a diff fn

Reply:
yeah the `throw new ErrorBadRequest()` checks prob should be outside this helper function

Reply:
also if we are gonna export these, would be best to make them very generic and move them to apiSigning

Reply:
moved these fns to apiSigning. I don't think we need to make this fn generic because the original signing and recover methods are generic enough to cover the generic case